test/intl402/DateTimeFormat/prototype/formatRange/this-is-not-object-throws.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google, Inc.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description:
+  Throws a TypeError if this is not Object.
+info:
+  Intl.DateTimeFormat.prototype.formatRange ( startDate , endDate )
+
+  1. Let dtf be this value.
+  2. If Type(dtf) is not Object, throw a TypeError exception.
+
+features: [Intl.DateTimeFormat-formatRange, Symbol]
+---*/
+
+var formatRange = Intl.DateTimeFormat.prototype.formatRange;
+
+assert.throws(TypeError, function() {
+  formatRange.call(undefined, undefined);
```

Comment:
I agree - I don't think we should be passing second values.

It would be fine to pass value timestamps or to pass `... undefined, undefined)` for both times (then we would get a RangeError instead of a type error if this test fails).

Reply:
sorry my bad. corrected now